features/fixtures/mazerunner/cxx-scenarios-bugsnag/src/main/java/com/bugsnag/android/mazerunner/scenarios/ExitInfoScenario.kt

```diff
@@ -0,0 +1,28 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import android.os.Handler
+import android.os.Looper
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+
+class ExitInfoScenario(
+    config: Configuration,
+    context: android.content.Context,
+    eventMetadata: String?
+) : Scenario(config, context, eventMetadata) {
+    external fun crash(value: Int): Int
+    override fun startScenario() {
+        super.startScenario()
+        Bugsnag.startSession()
+        val main: android.os.Handler = android.os.Handler(android.os.Looper.getMainLooper())
+        main.postDelayed(
+            object : java.lang.Runnable {
+                override fun run() {
+                    crash(2726)
+                }
+            },
+            500
+        )
+    }
```

Comment:
This scenario should rather throw a normal exception instead of relying on an unsatisfied link error